content/en/apps/guides/hosting/4.x/migrating-projects-docker-compose-to-k3s-on-vmware.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Migrating projects on VMware that were installed via docker-compose to k3s"
+linkTitle: "migrating projects on VMware from docker-compose to k3s"
+weight: 20
+description: >
+  Here we will outline the process for migrating projects that were installed on VMware by docker-compose on the VM's root disk to a k3s installation that exists in the same VMware datacenter.
+---
+
+### Prerequisites 
+
+This doc will assume you have followed or read over our guides to [self-hosting-k3s-multinode.md]() and [troubleshooting-k3s-on-vmware.md](). Please ensure your terminal is authenticated to your vCenter datacenter by following the govc or curl authentication steps in those docs.
+
+### Current cht-core installation setup
+
+This doc assumes your existing cht-core projects are installed on separate VM's that are launched by a VM template in VMware. Docker compose was used to install cht-core project on each VM, and all data was stored on that VMs root disk. There was also a VMware encyption policy that all above disks mentioned are encrypted.
+
+### Our goal cht-core setup
+
+Our goal is to merge all of our projects into a k3s VMware deployment for ease of maintainability, support of multi-node cht-core projects, and other benefits that are highlighted in our k3s VMware installation docs.
+
+To reach our goal, we will need to complete the following steps:
+* Shutting down existing running project, this will mean downtime for the project
+* Copy disk from shut-down VM
+* Mounting disk to k3s control-plane node
+* Decrypting disk
+* Copying data via rsync from old disk to a container inside k3s that uses PersistentVolumeClaims
+* Using the above PVC inside new cht-core deployment templates to re-deploy the cht-core project inside k3s
+* Updating nginx to forward traffic to our project installation
+* Verifying all services are running for cht-core
+* Taking a backup and verifying backup policy works 
+
+### Shut down existing project VM
+
+Identify which VM you wish to shutdown, by navigating and identifying the VM name that matches your project.
+
+      govc ls /<datacenter_name>/vm/<sub_dir>
+
+*Note* Before powering off the vm, ensure you have access to the vm's terminal to decrypt the root disk if you ever need to re-use this VM, or restart this process.
+Power the vm off:
+      govc vm.power -off /<datacenter_name>/vm/<sub_dir>
+
+### Copy disk from shut-down VM
+
+We will want to copy the data from the recently shut-down VM above to a new vmdk disk file that we will mount onto our k3s control-plane server. Copying the disk will ensure we have a backup plan if anything goes wrong during our migration process. This will keep the original project data intact.
+
+      govc device.info -vm /<datacenter_name>/vm/<sub_dir>
+
+Example of above output:
```

Comment:
@Hareet I propose changing this to `Example *subset* of the above output highlighting the SAN disk`  to clarify that the output from this command is a lot more.